src/platform/i386/loader.S

```diff
@@ -109,7 +111,7 @@ smploader:
 
 	movw    %cs, %ax
 	movw    %ax, %ds
-	lgdtl   RELOCATE_ADDR_CS(smpgdtdesc) /* using cs as ds: idea taken from Quest */
+	lgdt    RELOCATE_ADDR_CS(smpgdtdesc) /* using cs as ds: idea taken from Quest */
```

Comment:
Just out of curiosity, any reason for this change?

Reply:
I cannot remember the detail about this. Sorry for this, I might test the difference later.

Reply:
`l` is just the size modifier.  Likely defaults to `l` on 32 bit anyway?